src/components/DocumentationTopic/DocumentationHero.vue

```diff
@@ -149,16 +155,20 @@ $doc-hero-icon-dimension: 250px;
     right: 0;
   }
 
-  // black overlay
+  // overlay
   &:after {
     background: $doc-hero-overlay-background;
-    opacity: 0.7;
+    opacity: 0.85;
     width: 100%;
     position: absolute;
     content: '';
     height: 100%;
     left: 0;
     top: 0;
+
+    @include prefers-dark {
+      opacity: 0.7;
+    }
```

Comment:
Why don't we include
```css
.theme-dark & {
```

in this property? In this intended to be different from light to dark mode user preference?